uportal-war/src/main/java/org/jasig/portal/jgroups/protocols/DAO_PING.java

```diff
@@ -101,17 +108,18 @@ public void destroy() {
             log.info("No PingDao set, returning empty set for current cluster members");
             return Collections.emptyList();
         }
-        
-        //Get the current cluster members
-        final Map<Address, PhysicalAddress> existing_mbrs = pingDao.getAddresses(clusterName);
 
         //Add our address to the store
         getAndSavePhysicalAddress(clusterName);
-        
+
+        //Get the current cluster members
+        final Map<Address, PhysicalAddress> existing_mbrs = pingDao.getAddresses(clusterName);
+
```

Comment:
I don't agree with the above change.  I think you want your list of existing members to be everyone BUT yourself.  You don't want to send messages to yourself.